build_runner/lib/src/generate/watch_impl.dart

```diff
@@ -62,7 +62,6 @@ Future<ServeHandler> watch(
 }) async {
   builderConfigOverrides ??= const {};
   packageGraph ??= new PackageGraph.forThisPackage();
-
```

Comment:
Unnecessary change?

Reply:
yup, had previously needed a change in this file before someone went and gave me merge conflicts 😝 

reverted